python/pyarrow/feather.py

```diff
@@ -66,8 +66,13 @@ def read_table(self, columns=None):
         table = Table.from_arrays(columns, names=names)
         return table
 
-    def read_pandas(self, columns=None, nthreads=1):
-        return self.read_table(columns=columns).to_pandas(nthreads=nthreads)
+    def read_pandas(self, columns=None, nthreads=None, use_threads=False):
+        if nthreads is not None:
+            warnings.warn("`nthreads` argument is ignored, "
+                          "pass `use_threads` instead", FutureWarning,
+                          stacklevel=2)
```

Comment:
If the user had passed `nthreads > 1`, then we should set `use_threads=True` here for backwards compatibility